libcloudforensics/providers/gcp/forensics.py

```diff
@@ -541,18 +539,18 @@ def IsolatePods() -> None:
     logger.info(
         'Creating deny-all NetworkPolicy for {0:s} '
         'workload...'.format(workload_id))
-    mitigation.IsolatePodsWithNetworkPolicy(k8s_cluster, workload_pods)
+    mitigation.IsolatePodsWithNetworkPolicy(cluster, workload_pods)
```

Comment:
if network policy is not enabled this will fail (raise error), do you check if network policy is enabled before calling IsolatePods, or maybe disable the option of IsolatingPod if network policy isn't enabled, to avoid failing